src/Relogify/Relogify.fs

```diff
@@ -71,37 +67,46 @@ module App =
     let selectOpponentTabIndex = 0
     let settingsTabIndex = 1
 
-    let initModel () =
+
+    let init () =
+        Routing.RegisterRoute("TestRoute", typeof<TestRoutingPage>)
+
         let applicationSettings = ApplicationSettings.getApplicationSettings ()
+        let opponentListModel, opponentListCmdMsgs = OpponentList.initModel applicationSettings
+        let cmdMsgs = opponentListCmdMsgs |> List.map OpponentListCmdMsg
+
         { SomeFlag = false
           SelectedTabIndex = if applicationSettings |> areSet then selectOpponentTabIndex else settingsTabIndex
-          OpponentListModel= OpponentList.initModel
+          OpponentListModel = opponentListModel
           AboutModel = About.initModel
           AddResultModel = AddResult.initModel
           ApplicationSettings = applicationSettings
-          SettingsModel = Settings.initModel applicationSettings }
-
-    let init () =
-        Routing.RegisterRoute("TestRoute", typeof<TestRoutingPage>)
-        initModel (), []
+          SettingsModel = Settings.initModel applicationSettings.CommunityName.IsSome }, cmdMsgs
 
     let update msg (model: Model) =
         match msg with
         | OpponentListMsg opponentListMsg ->
-            let opponentListModel, opponentListCmdMsgs = OpponentList.update model.OpponentListModel opponentListMsg
-            { model with OpponentListModel= opponentListModel }, opponentListCmdMsgs |> List.map OpponentListCmdMsg
+            match model.ApplicationSettings.PlayerName, model.ApplicationSettings.CommunityName with
+            | (Some _, Some communityName) ->
```

Comment:
What is the idea behind matching on `model.ApplicationSettings.PlayerName` while it is not used?

Reply:
Now that I look at it, `communityName` is not used either - replacing with `areSet` instead.